src/js/structs/Service.js

```diff
@@ -126,6 +126,15 @@ module.exports = class Service extends Item {
   getServiceStatus() {
     let {tasksRunning} = this.getTasksSummary();
     let deployments = this.getDeployments();
+    let queue = this.getQueue();
+
+    if (queue != null && queue.delay && queue.delay.overdue) {
```

Comment:
I'd like some documentation here, as it's really hard to distinguish the both conditions.